client/blocks/happychat/chat-panel/index.jsx

```diff
@@ -0,0 +1,76 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import {
+	blur,
+	focus,
+	closeChat,
+	minimizeChat,
+	minimizedChat,
+} from 'state/happychat/ui/actions';
+import { getCurrentUser } from 'state/current-user/selectors';
+import isHappychatMinimizing from 'state/happychat/selectors/is-happychat-minimizing';
+import isHappychatOpen from 'state/happychat/selectors/is-happychat-open';
+import HappychatClient from 'blocks/happychat/chat-client';
+import { LAYOUT_PANEL_MAX_PARENT_SIZE } from 'blocks/happychat/chat-client/constants';
+
+/*
+ * Main chat panel UI component
+ */
+export class HappychatPanel extends Component {
+	static propTypes = {};
```

Comment:
Nit: we can remove this and `defaultProps` if nothing is inside.

Reply:
Sure :)